### ISSUE TITLE
Freezes when adding price data preferences under manjaro linux

### ISSUE THREAD

**Issue by TheCrazyInsanity** [Jacudibu/PRUNner#31]
When under the base planner and pressing the add price data preferences button the program freezes.

**Comment by Jacudibu** [comment]
Hey! Do you have any special base set-ups or does this also occur if you rename the UserData folder and just create an empty base on some random planet like for example Montem?

**Comment by TheCrazyInsanity** [comment]
> Hey! Do you have any special base set-ups or does this also occur if you rename the UserData folder and just create an empty base on some random planet like for example Montem?
> 
> Edit: I've also just published a new version, maybe that already fixes it for some magical reason?

i have a simple 3 building base on gibson

**Comment by louhy** [comment]
Just a comment, Linux Mint - no issues here with setting a preferred Exchange price preference.

**Comment by Jacudibu** [comment]
Yeah, also no issues here on Xubuntu. Might need more details about the base setup to reproduce it, can't believe something like that could be OS specific.
